src/rust/engine/process_executor/src/main.rs

```diff
@@ -99,6 +99,13 @@ fn main() {
         .multiple(true)
         .help("Environment variables with which the process should be run."),
     )
+      .arg(
+        Arg::with_name("jdk")
+            .long("jdk")
+            .takes_value(true)
+            .required(false)
+            .help("Symlink a JDK from .jdk in the working directory. For local execution, symlinks to the value of this flag. For remote execution, just requests that some JDK is symlinked if this flag has any value.")
```

Comment:
I'm not sure I like the fact that we only care about the value of the flag when we run locally, and ignore it when we remote. But, I can't think of a better way to do it, so it will probably have to be this way.

Reply:
Worth linking to the github issues we have about replacing this situation in as many places as possible.